.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Reference: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+
+# By default we assume that the novu-devs should review most changes
+@novuhq/novu-devs
```

Comment:
This doesn't match out offsite team agreement. There is a proposal by @LetItRock to implement something similar to this but we still haven't debated it.
I recommend its removal for now.